File: app/src/main/java/com/dmi/perfectreader/style/StyleLength.java
package com.dmi.perfectreader.style;

public class StyleLength {
    private float value;
    private Type type;

    // См. http://www.w3schools.com/cssref/css_units.asp
    public enum Type {
        // Абсолютные
        CM, MM, IN, PX, PT, PC,

        // Относительные
        EM, EX, CH, REM, VW, VH, VMIN, VMAX,

        // Зависят от контекста
        PERCENT
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/style/StyleLength.kt
package com.dmi.perfectreader.style

class StyleLength(val value: Float, val type: Type) {
    // См. http://www.w3schools.com/cssref/css_units.asp
    enum class Type {
        // Абсолютные
        CM,
        MM, IN, PX, PT, PC,

        // Относительные
        EM,
        EX, CH, REM, VW, VH, VMIN, VMAX,

        // Зависят от контекста
        PERCENT
    }
}
